### DISCUSSION TITLE
HRtoVRChat: Galaxy Watch support.

### DISCUSSION THREAD

**Discussion by DrIntelligent** [200Tigersbloxed/UnityMods#2] [Ideas]
Would it be possible to support the Galaxy watch?
Was curious to mess with this feature. Don't know if It's something I would use constantly afterwards, but it seemed interesting to try.

Like how the HRCounter mod for Beat Saber does here:
https://github.com/qe201020335/HRCounter#galaxy-watch

It can read in two ways. Via text file, or directly by grabbing information from the desktop app.

> 1. Use [HeartRateToWeb](https://github.com/loic2665/HeartRateToWeb) and [this](https://galaxystore.samsung.com/geardetail/tUhSWQRbmv) app on your watch.
> 2. Follow the instrucrion on [their github page](https://github.com/loic2665/HeartRateToWeb) to set up heart rate broadcast. Make sure you download the [first](https://github.com/loic2665/HeartRateToWeb/releases/tag/v1.0.0) realease, the csharp one is bugged.
> 3. Use [from file](https://github.com/loic2665/HeartRateToWeb#from-file) or [from /hr endpoint](https://github.com/loic2665/HeartRateToWeb#from-hr-endpoint) in the [OBS](https://github.com/loic2665/HeartRateToWeb#obs) section of HeartRateToWeb instructions.
> 4. If you decide to use "from file", then use the path to the hr.txt file generated by HeartRateToWeb as file:///PATH/TO/THE/FILE.
> 5. If you decide to use "/hr endpoint", then enter the feedlink as http://YOUR_IP:YOUR_PORT/hr

> For example: `"FeedLink": "file:///D:/example/folder/hr.txt",` and `"FeedLink": "http://192.168.1.100:6547/hr",`

**Comment by 200Tigersbloxed** [comment]
Alright sure! Just be be correct, the Galaxy Watches **don't** support Pulsoid or Hyperate, correct? (Pulsoid to be added v1.3.0)

**Reply by DrIntelligent** [reply]
Hey, thanks for the quick reply :D
To my knowledge Pulsoid and Hyperate apps are not available in a TizenOS variant/in the Galaxy Watch store. There's a chance the newly released Galaxy watch 4 might be able to use Hyperate since its software is a WearOS base + Tizen skin mix (from what I understand). I don't own this new watch though..

**Reply by 200Tigersbloxed** [reply]
Alright, I'll look into it and try and add it! (there's the third new support for v1.3.0 lol)

**Comment by 200Tigersbloxed** [comment]
Added in v1.3.0
